### DISCUSSION TITLE
Checks interface for GitHub Pages disappeared after .nojekyll

### DISCUSSION THREAD

**Discussion by smith558** [community/community#22508] [Other Feature Feedback, Questions, & Ideas]
<p>I’ve temporarily had a <code> .nojekyll</code> file in the root of my GitHub pages repository. (may not actually be related to the issue) <strong>I have deleted it.</strong> Afterwards, the GH default pages <code>pages-build-deployment</code> workflow is working in the background as expected but the floating Checks interface for the workflow that would have been shown next to commit names in the branch commits history has disappeared. (see gif below for what I mean by the “floating Checks interface”)</p>
<p><img src="https://user-images.githubusercontent.com/44023416/181071554-0b5c9216-4431-45e3-a1a3-0c2edd7ff7f9.gif" alt="checks" data-base62-sha1="1tqUmN5urTAa3DDBMu4GbYq444G" width="517" height="271" class="animated"></p>
<p>Is anyone able to help? I’ve found them pretty useful as indicators.</p>
<p>Thanks!<br>
Stanley</p>

**Comment by smith558** [comment]
<p><strong>BEFORE:</strong><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/0/8058f40fec70814617e8529ff151f33dadc2dab3.png" data-download-href="/uploads/short-url/ijpHgQ7u0TK4wHYkNqEXd8LcUqT.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/44023416/181071555-446a500f-b785-4fc6-8633-7b8ef696f538.png" alt="image" data-base62-sha1="ijpHgQ7u0TK4wHYkNqEXd8LcUqT" width="345" height="235" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/0/8058f40fec70814617e8529ff151f33dadc2dab3_2_345x235.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/0/8058f40fec70814617e8529ff151f33dadc2dab3_2_517x352.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/0/8058f40fec70814617e8529ff151f33dadc2dab3_2_690x470.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/0/8058f40fec70814617e8529ff151f33dadc2dab3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1258×858 129 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by smith558** [comment]
<p><strong>NOW:</strong><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/6/764177b5b63adab2b185af7e24a5dfe2aeb9d0a8.png" data-download-href="/uploads/short-url/gS8Bwwi5JLKW0WrKEA0C6ZzkjJ6.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/44023416/181071556-adca397b-a7b2-4844-b112-d5467fbb761b.png" alt="image" data-base62-sha1="gS8Bwwi5JLKW0WrKEA0C6ZzkjJ6" width="344" height="209" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/764177b5b63adab2b185af7e24a5dfe2aeb9d0a8_2_344x209.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/764177b5b63adab2b185af7e24a5dfe2aeb9d0a8_2_516x313.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/764177b5b63adab2b185af7e24a5dfe2aeb9d0a8_2_688x418.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/764177b5b63adab2b185af7e24a5dfe2aeb9d0a8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1240×752 83.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by yoannchaudet** [comment]
<p>Hello!</p>
<p>This is not the result of you removing a file but us moving your public Pages build to Actions. You can see the whole status of a build and deployment (including logs) in the Actions tab of your repository.</p>
<p>We did not re implement the status checks yet (which is why they are missing). This is actually being worked on now and should be available shortly.</p>

**Comment by smith558** [comment]
<p><a class="mention" href="/yoannchaudet">@yoannchaudet</a>  Thanks for your reply! Much appreciated.</p>
<p>Upon closer inspection, I actually realised Jekyll has nothing to do with the issue but could not find the source nonetheless. You have saved me time trying to figure this out.</p>
<p>Happy to see them coming back soon!</p>

**Comment by yoannchaudet** [comment]
<p>Happy to report this just got deployed to production a few minutes ago :sweat_smile: The checks should be back!</p>
